test/dwd_dynam.jl

```diff
@@ -310,7 +310,76 @@ end
     @test readout(m, vcat(u1, u2, u3), nothing, 0) == [u1 + u2, u2, u3]
     @test eval_dynamics(m, vcat(u1, u2, u3), [x1, x2], nothing, 0) == vcat(x1, u2 + x1 + 2*u2 + u3, x2)
   end
-end
 
+  @testset "VectorInterface for InstantaneousContinuousMachine" begin
+    m = InstantaneousContinuousMachine{Float64,3}(
+      1, 3, 1,
+      (u, x, p, t) -> u*1.5, # dynamics
+      (u, x, p, t) -> u+x, # readout
```

Comment:
I'm surprised this works,  since `u` is a vector of length 3 and `x` is a vector (of length 1) of vectors of length 3.

Reply:
fixed.